cloud/scope/powervs_machine.go

```diff
@@ -1069,7 +1069,7 @@ func (m *PowerVSMachineScope) CreateVPCLoadBalancerPoolMember() (*vpcv1.LoadBala
 				ID: loadBalancer.ID,
 			})
 			if err != nil {
-				return nil, fmt.Errorf("failed to fetch VPC load balancer details with ID: %s error: %v", *loadBalancer.ID, err)
+				return nil, fmt.Errorf("failed to fetch VPC load balancer details with ID: %s error: %v", *lbID, err)
```

Comment:
Uncovered a bug while testing so fixed it here as its a small change. Unrelated to this PR 

This line was causing a panic when we fail to fetch a loadbalancer with the ID as `loadBalancer` will be nil when.